src/cljs/rems/administration/workflow.cljs

```diff
@@ -0,0 +1,138 @@
+
+(defn get-localized-value [field key language]
+  (key (first (filter (comp #{(name language)} :langcode)
+                      (:localizations field)))))
+
+(defn license-view [license language]
+  (into [:div.form-item
+         [:h4 (text-format :t.administration/license-field (get-localized-value license :title language))]
+         [inline-info-field (text :t.administration/title) (:title license)]]
+        (concat (for [localization (:localizations license)]
+                  [inline-info-field (str (text :t.administration/title)
+                                          " "
+                                          (str/upper-case (:langcode localization))) (:title localization)])
+                [[inline-info-field (text :t.administration/type) (:type license)]
+                 (when (:textcontent license)
+                   (case (:type license)
+                     "link" [inline-info-field (text :t.create-license/external-link) [:a {:target :_blank :href (:textcontent license)} (:textcontent license) " " [external-link]]]
+                     "text" [inline-info-field (text :t.create-license/license-text) (:textcontent license)]
+                     nil))]
+                (when (= "link" (:type license))
+                  (for [localization (:localizations license)]
+                    [inline-info-field (str (text :t.create-license/external-link)
+                                            " "
+                                            (str/upper-case (:langcode localization))) [:a {:target :_blank :href (:textcontent localization)} (:textcontent localization) " " [external-link]]]))
+                [[inline-info-field (text :t.administration/start) (localize-time (:start license))]
+                 [inline-info-field (text :t.administration/end) (localize-time (:end license))]])
+        ))
+
+(defn round-view [actors]
+  [:div.form-item
+   [:h4 (text-format :t.create-workflow/round-n (inc (:round (first actors))))]
+   (let [{approvers "approver" reviewers "reviewer"} (group-by :role actors)]
+     [:div
+      (when (seq approvers)
+        [inline-info-field (text :t.create-workflow/approvers) (str/join ", " (map :actoruserid approvers))])
+      (when (seq reviewers)
+        [inline-info-field (text :t.create-workflow/reviewers) (str/join ", " (map :actoruserid reviewers))])])])
+
+(defn rounds-view [actors language]
+  (let [rounds (vals (group-by :round actors))]
+    (when (seq rounds)
+      [collapsible/component
+       {:id "rounds"
+        :title (text :t.administration/rounds)
+        :top-less-button? (> (count rounds) 5)
+        :open? (<= (count rounds 5))
+        :collapse (into [:div]
+                        (for [round rounds]
+                          [round-view round]))}])))
+
+(defn licenses-view [licenses language]
+  [collapsible/component
+   {:id "licenses"
+    :title (text :t.administration/licenses)
+    :top-less-button? (> (count licenses) 5)
```

Comment:
The limit of 5 is now hard coded in quite many places, should it be a def somewhere?

Reply:
I don't think it's useful to put it anywhere yet.

There might be a case for a reusable collapse with this kind of functionality later. The difficulty is in that the regular collapse takes any component as child and does not know of any counts. Also the counts might actually be use case specific.